pkg/healthcheck/healthcheck.go

```diff
@@ -1370,6 +1383,19 @@ func (hc *HealthChecker) checkCanGet(namespace, group, version, resource string)
 	return hc.checkCanPerformAction("get", namespace, group, version, resource)
 }
 
+func (hc *HealthChecker) checkAPIService(serviceName string) error {
+	apiServiceClient, err := apiregistrationv1client.NewForConfig(hc.kubeAPI.Config)
+	if err != nil {
+		return err
+	}
+
+	_, err = apiServiceClient.APIServices().Get(serviceName, metav1.GetOptions{})
```

Comment:
Would it be helpful to check the `Available` status condition as well?

Reply:
`Available` status is in `Conditions` of type `[]v1.APIServiceCondition`.  I'm assuming there will be only one element in it. Is that correct?

Reply:
I don't know, but we can always iterate over the list of conditions until we find the `Available` condition.